tests/test_api.py

```diff
@@ -25,13 +25,16 @@ def test_template():
     assert api2.specification['info']['title'] == 'other test'
 
 
+
 def test_invalid_operation_does_stop_application_to_setup():
-    with pytest.raises(AttributeError):
-        Api(TEST_FOLDER / "fakeapi/op_error_api.yaml", "/api/v1.0",
-            {'title': 'OK'})
+    with pytest.raises(AttributeError) as exc_info:
+        Api(TEST_FOLDER / "fakeapi/op_error_api.yaml", "/api/v1.0", {'title': 'OK'})
+
+    assert 'fakeapi.module_with_error.something' in str(exc_info.value)
+    # TODO: I would be nice to show the root cause or the import error of 'module_with_error' like this ?
```

Comment:
I was not able to get the traceback to to original cause, so I added this TODO comment, maybe better to make a new issue ?
I stubled upon [this article](http://code.activestate.com/recipes/578252-python-exception-chains-or-trees/?in=user-4182236) with may come in handy here ?